x/gov/keeper/proposal.go

```diff
@@ -53,6 +53,21 @@ func (k Keeper) GetProposal(ctx sdk.Context, proposalID uint64) (types.Proposal,
 	return prop, true
 }
 
+func (k Keeper) GetProposals(ctx sdk.Context) ([]types.Proposal, error) {
```

Comment:
can you add a test for this one in keeper_test?

Reply:
I think is more optimal if you use the iterator for this based on the 

	ProposalsPrefix          = []byte{0x01}

Reply:
It's done